dto/src/main/java/org/jboss/pnc/dto/requests/validation/VersionValidationRequest.java

```diff
@@ -33,9 +34,15 @@
 @JsonDeserialize(builder = VersionValidationRequest.Builder.class)
 public class VersionValidationRequest {
 
+    /**
+     * Id of the product version.
```

Comment:
As I was the person that added this feature, I can specify it. 

"Id of the product version. The product version is used to prevent duplicate milestones with the same milestone version."